corehq/apps/reports/templates/reports/form/partials/single_form.html

```diff
@@ -128,26 +128,28 @@
     {% endblocktrans %}
 </div>
 {% endif %}
-<div class="pull-right">
-    {% if auth_context.authenticated %}
-        <i class="icon-lock"></i>
-        {% if user_info.id != auth_user_info.id %}
-        {% blocktrans with auth_user=auth_user_info|pretty_doc_info user=user_info|pretty_doc_info %}
-            Submitted by {{ auth_user }} on behalf of {{ user }}
-        {% endblocktrans %}
+<div class="clearfix">
+    <div class="pull-right">
```

Comment:
i was commenting just as you pushed a change, but I would move this block after the `nav-tabs` UL. It seems weird to have this right above it in the first place. You shouldn't need the `clearfix` div when you do this.